ouroboros-network/src/Ouroboros/Network/PeerSelection/State/KnownPeers.hs

```diff
@@ -149,6 +162,32 @@ invariant KnownPeers{..} =
         (Set.fromList (PSQ.keys nextConnectTimes)))
 
 
+-------------------------------
+-- KnownPeerInfo manipulation
+--
+
+alterKnownPeerInfo
```

Comment:
Add haddocks, maybe include a use case. Describe where the arguments are coming from.

Reply:
I would also move the definition after `alter` function, as they are related.